doc/api/globals.md

```diff
@@ -122,7 +122,6 @@ The `queueMicrotask()` method queues a microtask to invoke `callback`. If
 `callback` throws an exception, the [`process` object][] `'uncaughtException'`
 event will be emitted.
 
-In general, `queueMicrotask` is the idiomatic choice over `process.nextTick()`.
 `process.nextTick()` will always run before the microtask queue, and so
```

Comment:
FWIW, this whole paragraph is kinda sketchy. There is nothing unexpected about the nextTick execution order. As for Node 10 it is entirely predictable and defined, with no weird edge cases. It might not be "desired" but that's up to the individual user.

Reply:
I may be wrong, but the transition from the previous paragraph to the `process.nextTick()` mention seems a bit abrupt and unexpected now. Maybe we can add some smooth connection? Like `On the other hand...`, `` Similar `process.nextTick()`... `` etc?